docs/source/dev/imageseries-usage.rst

```diff
@@ -0,0 +1,58 @@
+.. _examples:
+
+Example of Imageseries Usage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Here is an example of how the imageseries is used. One thing we commonly do is
+to process the raw image files by adding flips and subtract off the background.
+Then we save it as a new imageseries. This example saves it into a HDF5 file,
+but it is more common to use the frame-cache (sparse matrices), which is way
+smaller.
+
+.. code-block:: python
+
+    import numpy as np
+
+    from hexrd import imageseries
+    from hexrd.imageseries.omega import OmegaWedges
+
+    # Inputs
+    DarkFile = 'dark-50pct-100f'
+    H5File = 'example.h5'
+    fname = 'example-images.yml'
+    mypath = '/example'
+
+    # Raw image series: directly from imagefiles
+    imgs = imageseries.open(fname, 'image-files')
+    print "number of frames: ", len(imgs),
+          "\ndtype: ", imgs.dtype,
+          "\nshape: ", imgs.shape
+
+    # Make dark image from first 100 frames
+    pct = 50
+    nf_to_use = 100
+    dark = imageseries.stats.percentile(imgs, pct, nf_to_use)
+    np.save(DarkFile, dark)
+
+
+    # Now, apply the processing options
+    ProcessedIS = imageseries.process.ProcessedImageSeries
```

Comment:
Might be more typical to just have at the top of the file `from imageseries.process import ProcessedImageSeries as ProcessedIS`.